spyder/plugins/editor/widgets/base.py

```diff
@@ -565,8 +561,8 @@ def get_cell_as_executable_code(self, cursor=None):
         if cursor is None:
             cursor = self.textCursor()
         ls = self.get_line_separator()
-        cursor, whole_file_selected = self.select_current_cell(cursor)
-        line_from, line_to = self.get_selection_bounds(cursor)
+        cursor, _ = self.select_current_cell(cursor)
+        line_from, _ = self.get_selection_bounds(cursor)

```

Comment:
```suggestion
        cursor, __ = self.select_current_cell(cursor)
        line_from, __ = self.get_selection_bounds(cursor)
```

It's better to use `__` for dummy variables when unpacking args to avoid shadowing our translation function by mistake.